docker-compose.yml

```diff
@@ -82,5 +82,12 @@ services:
     ports:
       - 8005:8005
 
+  collaboration-service:
+    container_name: collaboration-service
+    build: ./collaboration-service
+    restart: unless-stopped
+    ports:
+      - 8003:8003
+
```

Comment:
Minor issue, put this between the services using port 8002 and 8004 for easier readability